src/bin/huak/commands/install.rs

```diff
@@ -6,16 +6,30 @@ use huak::ops;
 use huak::project::Project;
 
 /// Run the `install` command.
-pub fn run(all: bool) -> CliResult<()> {
+pub fn run(groups: Option<Vec<String>>, all: bool) -> CliResult<()> {
     let cwd = env::current_dir()?;
     let project = match Project::from(cwd) {
         Ok(p) => p,
         Err(e) => return Err(CliError::new(e, ExitCode::FAILURE)),
     };
 
-    if let Err(e) = ops::install::install_project_dependencies(&project, all) {
-        return Err(CliError::new(e, ExitCode::FAILURE));
-    };
+    match groups {
+        Some(opt_groups) => {
+            if let Err(e) = ops::install::install_optional_dependencies(
+                &project,
+                &opt_groups,
+            ) {
+                return Err(CliError::new(e, ExitCode::FAILURE));
+            };
+        }
+        None => {
+            if let Err(e) =
+                ops::install::install_project_dependencies(&project, all)
+            {
+                return Err(CliError::new(e, ExitCode::FAILURE));
+            };
```

Comment:
Wondering if we should just `install_project_dependencies(&project, &groups, &install_all)`. If we don't I want those install functions to have consistent naming. We're intentionally operating on a Project. So `install_optional_project_dependencies` might be verbose, but I think it's wise.

Reply:
Hmmm, yeah looks like extra function isn't necessary at the moment.